app/views/layouts/application.html.erb

```diff
@@ -5,39 +5,39 @@
     <div id="global-content-wrapper">
       <!-- turn off "global" navbar if directed to (e.g. on home page) -->
       <% if @navbar == false %>
-      
-        <%= bootstrap_flash %>
+
+        <%= render partial: 'shared/flash_helper', flash: flash %>
```

Comment:
Can we replace this with `<%= render 'shared/flash_helper', flash: flash %>`?
Less keystrokes :)